src/editor.c

```diff
@@ -1234,15 +1236,65 @@ editor_get_indent_prefs(GeanyEditor *editor)
 }
 
 
+static inline gint compare_line_indentation(ScintillaObject *sci, gint line1, gint line2)
```

Comment:
As said now we use C99, I don't mind, I'm just questioning the actual usefulness.  But if you like it also to hint the human reader "this is just splitting for readability", okay.